packages/screens/UserPublicProfile/UserPublicProfileScreen.tsx

```diff
@@ -211,7 +211,7 @@ export const UserPublicProfileScreen: ScreenFC<"UserPublicProfile"> = ({
   return (
     <ScreenContainer
       key={`UserPublicProfile ${id}`} // this key is to reset the screen state when the id changes
-      forceNetworkId={network?.id}
+      forceNetworkFeatures={[NetworkFeature.UPP]}
```

Comment:
this is wrong, we need to force the network of the current UPP user like done previously